DQM/L1TMonitor/interface/L1TStage2uGT.h

```diff
@@ -75,9 +75,14 @@ class L1TStage2uGT: public DQMEDAnalyzer {
    int algoBitLastBxInTrain_;
    const std::string algoNameFirstBxInTrain_;
    const std::string algoNameLastBxInTrain_;
-   
+ 
+   std::vector<std::string> unprescaledAlgoShortList_ ;
+   std::vector<std::string> prescaledAlgoShortList_; 
+
    // Booking of histograms for the module
-   
+   std::vector<std::pair<std::string,int>> unprescaledAlgoBit_;
```

Comment:
Since these hold a pair made of algo name + algo bit the variable names are misleading.